test/validateOptions.js

```diff
@@ -13,14 +13,14 @@ describe('Plugin option validation', function() {
     });
     describe('Given two options', function(){
        it('Should return an error', function(){
-           plugin.register(null, {lower_case: true, camel_case: true}, function(err) {
+           plugin.register(null, {lowerCase: true, camelCase: true}, function(err) {
```

Comment:
could you write some tests where you pass in a joi schema to `camelCase`, and make sure it validates?